### PR TITLE
[PHP/laravel] Add PRipple coroutine engine for Laravel framework

### PR DESCRIPTION
**Pull request by cclilshy**
The pripple coroutine engine was added to the composite test of the laravel framework, The relevant files have passed the test of my machine,  thank

### CODE REVIEW

File: frameworks/PHP/laravel/benchmark_config.json [hunk 1]
```diff
@@ -138,6 +138,29 @@
...
+			"database": "MySQL",
+			"framework": "laravel",
+			"language": "PHP",
+			"flavor": "PHP8.3",
+			"orm": "Full",
+			"platform": "pripple",
+			"webserver": "PServer",
+			"os": "Linux",
+			"database_os": "Linux",
+			"display_name": "laravel",
```
Comment (joanhey) [inline frameworks/PHP/laravel/benchmark_config.json:161]: Please change `"display_name":` to  `"laravel [pripple]"`.
So I'll be easily identifiable in the visualize graphs, and not confunded with the plain Laravel permutation.

File: frameworks/PHP/laravel/laravel-pripple.dockerfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+COPY --link . .
+
+RUN echo "PRP_HTTP_LISTEN=http://0.0.0.0:8080" >> .env
+RUN echo "PRP_HTTP_COUNT=64" >> .env
+
+RUN mkdir -p bootstrap/cache
+RUN mkdir -p storage/logs
+RUN mkdir -p storage/framework/sessions
+RUN mkdir -p storage/framework/views
+RUN mkdir -p storage/framework/cache
```
Comment (joanhey) [inline frameworks/PHP/laravel/laravel-pripple.dockerfile:36]: Please run it all in one step.
```
RUN mkdir -p bootstrap/cache \
            storage/logs \
            storage/framework/sessions \
            storage/framework/views \
            storage/framework/cache

```

File: frameworks/PHP/laravel/laravel-pripple.dockerfile [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+RUN docker-php-ext-enable pdo_mysql opcache posix pcntl sockets
+RUN docker-php-ext-enable --ini-name zz-event.ini event
+RUN echo "opcache.enable_cli=1" >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
+RUN echo "opcache.jit=1205" >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
+RUN echo "opcache.jit_buffer_size=128M" >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
+
+# Initialize
+WORKDIR /laravel
+COPY --from=composer --link /usr/bin/composer /usr/local/bin/composer
```
Comment (joanhey) [inline frameworks/PHP/laravel/laravel-pripple.dockerfile:26]: Please, move the COPY composer  before or after the steps WORKDIR and COPY . .
Now it is in the middle.

**Review by joanhey — CHANGES_REQUESTED** [review]

**Comment by cclilshy** [comment]
Hello, I have adjusted and verified the operation according to the marked style. If there is nothing to add, I will declare it solved. Thank you again for your review

**Review by joanhey — APPROVED** [review]

**Pull request merged**